package.json

```diff
@@ -1,49 +1,11 @@
 {
   "name": "@newfold-labs/wp-module-migration",
-  "version": "1.2.3",
-  "description": "",
-  "main": "build/index.js",
+  "description": "The migration module is used to initiate the migration process by installing the required plugins for migration.",
+  "license": "GPL-3.0-or-later",
+  "private": true,
   "scripts": {
-    "build": "wp-scripts build ./src/index.js --config ./scripts/webpack.config.js",
-    "lint": "wp-scripts lint-js ./src/index.js",
-    "prepublishOnly": "npm run build",
-    "set-version-bump": "node ./set-version-bump.js && composer install && npm i && npm run build && composer run i18n",
-    "start": "wp-scripts start ./src/index.js --config ./scripts/webpack.config.js"
+    "set-version-bump": "node ./set-version-bump.js && composer i && composer run i18n"
```

Comment:
can also remove this script since we're removing the file it references.

Reply:
Thanks, @circlecube! I just thought about it and went ahead and removed both `package` files since they weren’t being used anyway.